### ISSUE TITLE
Stats: The number of Followers displayed under Stats doesn't always match the real number of .com + Email Followers

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/wp-calypso#11233]
The number of Followers displayed under Stats doesn't always match the real number of .com + Email Followers.

#### Steps to reproduce
1. Starting at URL: https://wordpress.com/stats/day/
2. View the number of your site's followers in the upper right hand corner. You can see, from the following screenshot, that the number of followers is 90 for my site:

<img width="1074" alt="screen shot 2017-02-07 at 13 19 43" src="https://cloud.githubusercontent.com/assets/2998162/22692732/3b07cf8a-ed38-11e6-9d4a-e1e630c78833.png">

3. Click on the "Followers" button to get more details and look at the number of .com vs. Email Followers.

#### What I expected

I expected the number of .com + Email Followers to add up to the number displayed on the stats page (in my case, 90).

#### What happened instead

I saw that I had 92 .com Followers and 1 Email Follower:

<img width="792" alt="screen shot 2017-02-07 at 13 25 21" src="https://cloud.githubusercontent.com/assets/2998162/22692894/e8ab9c16-ed38-11e6-9fd2-40f67f17fd32.png">

<img width="783" alt="screen shot 2017-02-07 at 13 26 05" src="https://cloud.githubusercontent.com/assets/2998162/22692917/01eaee52-ed39-11e6-98b8-81493195ac8e.png">

The total that I would expect to see under Stats is therefore 93, not 90. 

The reason for this is not at all obvious to me: Perhaps partly related to #2780? I am following my blog both via the Reader and Email. That would still come to 92, not 90, though.

#### Browser / OS version

Chrome

#### Context / Source

#user-report  #manual-testing

**Comment by SiobhyB** [comment]
This was originally reported in 3049553-t.

**Comment by designsimply** [comment]
I tested and was unable to reproduce. 

My test blog account (design5279) shows 13 followers in the header at https://wordpress.com/stats/day/design5279.wordpress.com with 12 at https://wordpress.com/people/followers/design5279.wordpress.com and 1 at https://wordpress.com/people/email-followers/design5279.wordpress.com

I also checked the account from the original report of the problem (giovannineri) and found 84 followers listed in the header at https://wordpress.com/stats/day/kurvenal.com which correctly adds up from 14 at https://wordpress.com/people/followers/kurvenal.com and 70 at https://wordpress.com/people/email-followers/kurvenal.com

@SiobhyB, may I ask which account/site you used to make the screenshots in this bug report and would you be willing to check on those numbers again and let me know if they're still off?

**Comment by SiobhyB** [comment]
@designsimply: The screenshots were taken from my own account (siobhyb) and site (siobhan.blog).

I took another look and the correct number of followers (93 .com followers + 1 email follower) is now displaying on the Stats page:

<img width="1075" alt="screen shot 2017-02-23 at 19 39 38" src="https://cloud.githubusercontent.com/assets/2998162/23275842/ee8a8178-f9ff-11e6-9a17-3e598ac54292.png">

It looks like this got fixed somehow, somewhere. 🎉 :) Closing.
